src/hooks/useReleaseStatus.ts

```diff
@@ -8,20 +8,23 @@ export const useReleaseStatus = (release: ReleaseKind) => {
       return runStatus.Unknown;
     }
 
-    const progressing = release.status.conditions.some((c) => c.reason === 'Progressing');
-    if (progressing) {
-      return runStatus['In Progress'];
-    }
-
-    const succeeded = release.status.conditions.every(
-      (c) => c.reason === 'Succeeded' && c.status === 'True',
+    const succeeded = release.status.conditions.some(
+      (c) =>
+        c.type === ReleaseCondition.Released && c.reason === 'Succeeded' && c.status === 'True',
```

Comment:
just extract the ReleasedCondition once and you can use the same condition through out the function. No need to traverse the conditions again.
```suggestion
    const releasedCondition = release.status.conditions.find(
      (c) =>
        c.type === ReleaseCondition.Released
```